tests/test_api.py

```diff
@@ -77,6 +79,33 @@ def test_represent(self):
 
         logger.info("✅ representation api test is done")
 
+    def test_represent_encoded(self):
+        with open(image_path, "rb") as image_file:
```

Comment:
what if image_path?

you may consider to set it to "dataset/img1.jpg"

Reply:
Sure that's the main problem